easybuild/easyconfigs/a/AlphaFold/AlphaFold-2.3.0-foss-2022a-CUDA-11.7.0.eb

```diff
@@ -0,0 +1,167 @@
+easyblock = 'PythonBundle'
+
+name = 'AlphaFold'
+version = '2.3.0'
+versionsuffix = '-CUDA-%(cudaver)s'
+
+homepage = 'https://deepmind.com/research/case-studies/alphafold'
+description = "AlphaFold can predict protein structures with atomic accuracy even where no similar structure is known"
+
+toolchain = {'name': 'foss', 'version': '2022a'}
+
+builddependencies = [
+    # required for installing dm-tree
+    ('Bazel', '5.1.1'),
+    ('CMake', '3.23.1'),
+]
+
+dependencies = [
+    ('Python', '3.10.4'),
+    ('CUDA', '11.7.0', '', SYSTEM),
+    ('SciPy-bundle', '2022.05'),
+    ('PyYAML', '6.0'),
+    ('TensorFlow', '2.11.0', versionsuffix),
+    ('Biopython', '1.79'),
+    ('HH-suite', '3.3.0'),
+    ('HMMER', '3.3.2'),
+    ('Kalign', '3.3.5'),
+    ('jax', '0.3.14', versionsuffix),  # also provides absl-py
+    ('UCX-CUDA', '1.12.1', versionsuffix),
+    ('cuDNN', '8.4.1.50', versionsuffix, SYSTEM),
+    ('NCCL', '2.12.12', versionsuffix),
+    # AlphaFold doesn't work with latest OpenMM, so have to use v7.5.1
+    ('OpenMM', '7.5.1', '-DeepMind-patch'),
```

Comment:
There's guidelines in https://github.com/deepmind/alphafold/issues/404 on how to make AlphaFold compatible with OpenMM 7.7.0, it's probably worth following those?

In addition, I think it's important that a CUDA-aware OpenMM is used as dependency (and that's unfortunately not trivial to add, due to hitting internal compiler errors, see https://github.com/vscentrum/vsc-software-stack/issues/118

Reply:
I saw that solution but was hesitant to apply it, as it was not added to the release. I'll give it a try.